baseapp/abci.go

```diff
@@ -364,6 +364,9 @@ func (app *BaseApp) halt() {
 
 // snapshot takes a snapshot of the current state and prunes any old snapshottypes.
 func (app *BaseApp) snapshot(height int64) {
+	if app.snapshotManager == nil {
+		return
```

Comment:
Hm, this should possibly error - this will only be called if the user has configured snapshotting by setting `snapshot_interval`, in which case they may wonder why it isn't working.

Reply:
ACK